contributors/design-proposals/node/troubleshoot-running-pods.md

```diff
@@ -45,26 +45,25 @@ A solution to troubleshoot arbitrary container images MUST:
 *   fetch troubleshooting utilities at debug time rather than at the time of pod
     creation
 *   be compatible with admission controllers and audit logging
-*   allow discovery of debugging status
+*   allow discovery of current debugging status
 *   support arbitrary runtimes via the CRI (possibly with reduced feature set)
 *   require no administrative access to the node
 *   have an excellent user experience (i.e. should be a feature of the platform
     rather than config-time trickery)
-*   have no *inherent* side effects to the running container image
+*   have no _inherent_ side effects to the running container image
+*   v1.Container must be available for inspection by admission controllers
 
 ## Feature Summary
 
 Any new debugging functionality will require training users. We can ease the
 transition by building on an existing usage pattern. We will create a new
 command, `kubectl debug`, which parallels an existing command, `kubectl exec`.
-Whereas `kubectl exec` runs a *process* in a *container*, `kubectl debug` will
-be similar but run a *container* in a *pod*.
+Whereas `kubectl exec` runs a _process_ in a _container_, `kubectl debug` will
```

Comment:
i think this is underdocumented here https://kubernetes.io/docs/tasks/debug-application-cluster/get-shell-running-container/ that kubectl exec runs a process in an existing container